tests/foreman/api/test_http_proxy.py

```diff
@@ -303,6 +303,11 @@ def test_positive_sync_proxy_with_certificate(request, target_sat, module_org, m
 
     :customerscenario: true
     """
+
+    @request.addfinalizer
+    def _finalize():
+        target_sat.custom_certs_cleanup()
```

Comment:
Can we keep only one request.addfinalizer method in this test?

Reply:
that kind of contradicts with other concerns voiced in comments, if there are more elements to remove and you want to make sure all are marked to be finalized immediately after creation but not before, then you end up with multiple finalizers

Reply:
I didn't understand this, do we really need to finalizers calling `target_sat.custom_certs_cleanup()` this specific test?
Initially, I would've held the merge on this, but now that it's self-merged, could you please take care of this in a separate PR if you agree with the above?

Reply:
@pondrejk ^